src/app/notebox/page.tsx

```
@@ -0,0 +1,75 @@
+  const [isModalOpen, setIsModalOpen] = useState(false);
+  const { data: loginUser } = useUserData();
+  const [notes, setNotes] = useState<any[]>([]);
+
+  console.log('스토어 유저', loginUser);
+  useEffect(() => {
+    const getUserNotes = async () => {
+      try {
+        const userId = await fetchUser(); // 로그인한 사용자 ID
+        // console.log('✅ 로그인된 사용자 ID:', userId);
+
+        const userInfo = await fetchUserInfo(userId); // 사용자 정보
```

ESLint

[error] 'userInfo' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)